src/plugins/common/format/lib/format-plugin.js

```diff
@@ -710,10 +710,10 @@ define('format/format-plugin', [
 	/**
 	 * register the plugin with unique name
 	 */
-	return Plugin.create('format', {
-		config: DEFAULT_CONFIG,
+	var plugin = Plugin.create('format', {
+		buttonConfig: structuredClone(DEFAULT_BUTTON_CONFIG),
```

Comment:
Cite/Quote, or some plugin adds some elements to `config`, must be changed to `buttonConfig`

Reply:
Done